docs/src/starknet/101.md

```diff
@@ -0,0 +1,209 @@
+# `sncast` 101
+
+This page showcases the standard usage of various `sncast` commands.
+For more in-depth explanation, please follow other guides from "sncast Overview".
+
+## Create and Deploy an Account
+
+First, create an account.
+This generates the account details but doesn't put it on the network yet.
+
+<!-- { "ignored": true } -->
+```shell
+$ sncast account create \
+  --name my_account \
+  --network sepolia
+```
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: account create
+add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
+address: 0x[..]
+max_fee: [..]
+message: Account successfully created. Prefund generated address with at least <max_fee> STRK tokens. It is good to send more in the case of higher demand.
+
+After prefunding the address, run:
+sncast account deploy --network sepolia --name my_account
+```
+
+</details>
+
+> 📝 **Note**
+> The `address` in the output has been redacted to avoid confusion.
+> It will be visible when running the command locally.
+
+After creating an account, send enough of STRK tokens to the address to cover the price of the
+account creation.
+At least `max_fee` amount of fri (1 STRK = 10<sup>18</sup> fri) should be sent.
+
+> 💡 **Tip**
+> On Sepolia, [this free faucet](https://starknet-faucet.vercel.app/) can be used to fund the account.
+
+
+Then, deploy the account using the command that was provided in output, below `After prefunding the address, run`:
+
+<!-- { "ignored": true } -->
+```shell
+$ sncast account deploy \
+  --network sepolia
+  --name my_account
+```
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: account deploy
+transaction_hash: 0x[..]
+
+
+To see invocation details, visit:
+transaction: https://sepolia.starkscan.co/tx/0x[..]
+```
+
+</details>
+
+[Read more about accounts here](./account-import.md)
+
+> 💡 **Tip**
+> Existing accounts can be imported to `sncast` as well. See [Importing Accounts](./account-import.md) for more
+> information.
+
+## Calling a Contract
+
+Simply calling a contract doesn't require an account
+
+```shell
+$ sncast call \
+  --contract-address 0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d \
+  --function decimals \
+  --network sepolia
+```
```

Comment:
Maybe use [this example](https://foundry-rs.github.io/starknet-foundry/starknet/call.html#general-example) (it calls `get_balance` function? Calling decimals functions seems not very common.

Reply:
I think calling strk token contract makes sense for this kind of example

Reply:
But maybe I can change it to some other method